zap/src/main/java/org/zaproxy/zap/extension/history/HistoryFilterPlusDialog.java

```diff
@@ -494,6 +540,84 @@ private JScrollPane getConfidenceScroller() {
         return confidenceScroller;
     }
 
+    private JScrollPane getUrlRegxSamples() {
+        if (urlRegxSamples == null) {
+            // TODO save the list in better place
+            final String[] URL_REGX_INC_LIST = {
+                ".*\\.(js|css|woff2|png|jpeg|jpg|gif|svg)$", ".*\\.(html|asp|jsp|php)$"
```

Comment:
Not sure why I didn't notice this earlier. But doing extensions with an ending anchor will mean that URLs with params are not matched by the filter. ex: https://example.com/foo/bar.php?baz=lmnop